src/interface.jl

```diff
@@ -123,7 +124,9 @@ end
 struct SingularJacobianException{B} <: Exception where {B<:Bijector}
     b::B
 end
-Base.showerror(io::IO, e::SingularJacobianException) = print(io, "jacobian of $(e.b) is singular")
+function Base.showerror(io::IO, e::SingularJacobianException)
+    print(io, "jacobian of $(e.b) is singular")
+end
```

Comment:
Is this for line character limit?

Reply:
If so I suggest to do the following
```julia
Base.showerror(io::IO, e::SingularJacobianException) = 
    print(io, "jacobian of $(e.b) is singular")

```

Reply:
Yup!
But does that follow the [style-guide](https://turing.ml/dev/docs/contributing/style-guide#method-definitions): "Only use short-form function definitions when they fit on a single line"?

Reply:
Opps. I think it is me who is not following the style guide.

Reply:
Let's stick to the guide then. Thanks for pointing out this.